cmd/gopcomm/community_yap.gox

```diff
@@ -489,6 +489,20 @@ post "/upload", ctx => {
 	community.UploadFile(ctx)
 }
 
+get "/retryCaptionGenerate", ctx => {
```

Comment:
This URI seems doesn't meet bast practice. Refer https://www.oreilly.com/library/view/restful-web-services/9780596809140/ch04.html